src/App/Pages/Vault/ScanPage.xaml.cs

```diff
@@ -55,6 +52,53 @@ public ScanPage(Action<string> callback)
         protected override void OnAppearing()
         {
             base.OnAppearing();
+            StartScanner();
+        }
+
+        protected override void OnDisappearing()
+        {
+            StopScanner();
```

Comment:
Add `FireAndForget()` otherwise if an exception happens is swallowed by the `Task` and we don't have the feedback